lib/irb/context.rb

```diff
@@ -146,9 +146,21 @@ def initialize(irb, workspace = nil, input_method = nil)
         @newline_before_multiline_output = true
       end
 
-      @command_aliases = IRB.conf[:COMMAND_ALIASES]
+      @user_aliases = IRB.conf[:COMMAND_ALIASES]
+      @command_aliases = @user_aliases.merge(KEYWORD_ALIASES)
```

Comment:
This used to work, but it does not now.
```ruby
irb(main):001> IRB.conf[:COMMAND_ALIASES][:ss] = :show_source
irb(main):002> ss Set#include?
```
It does not work when you write it directly into IRB console, but show_cmds shows not-working aliases.
```
Aliases
  $              Alias for `show_source`
  @              Alias for `whereami`
  ss             Alias for `show_source`
```
The problem is that when `IRB.conf[:COMMAND_ALIASES]` change, `@user_aliases` will change (because referencing same Hash instance) but `@command_aliases` does not change.

I think one of these is better
```ruby
# Use a snapshot of COMMAND_ALIESES.
# Both `@user_alias` and `@command_aliases` does not change when `IRB.conf[:COMMAND_ALIASES]` change.
@user_aliases = IRB.conf[:COMMAND_ALIASES].dup
```

```ruby
# Both `@user_alias` and `command_aliases` change when `IRB.conf[:COMMAND_ALIAS]` change.
def command_aliases() = @user_aliases.merge(KEYWORD_ALIASES)
```